dired.sublime-settings

```diff
@@ -33,6 +33,13 @@
   //     "git_path": ""
   "git_path": "git",
 
+  // Path to hg executable.
+  // Wildcards are supported, e.g. if you use TortoiseHg it would be:
+  //     "hg_path": "%ProgramFiles%/TortoiseHg/hg.exe"
```

Comment:
1. `%var%` is being expanded (at least on windows machines) (tested in sublime console)

        >>> import os
        >>> import os.path
        >>> os.path.expandvars(os.path.expanduser("%ProgramFiles%"))
        'C:\\Program Files'

2. will fix

3. ok, thanks for help, i'm not that skilled in git